releases/mainnet/1.2.5/index.html

```diff
@@ -0,0 +1,24 @@
+<html>
+	<head>
+		<title>Mainnet Release 1.2.5</title>
+	</head>
+	<body>
+		<div>
+			<h2>
+				<strong>Mainnet Release 1.2.5</strong>
+			</h2>
+
+			<ul>
+				<li><a href="/releases/mainnet/1.2.5/contracts.json">Contract addresses</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmbVHXvPGPBNJoHLmaJ8CkYTVWXic7EextRvnkgRDLTCMa">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="/releases/mainnet/1.2.5/abi/index.html">ABIs</a></li>
+				<!-- <li>Links to GitHub repositories</li>
+					<ul>
+						DSS deploy scripts (<a href="https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.2.5" target="_blank">changelog</a>)</li>
```

Comment:
This can be uncommented now: https://github.com/makerdao/dss-deploy-scripts/releases/tag/1.2.5

Reply:
done